.github/workflows/workflow.yml

```diff
@@ -92,7 +92,9 @@ jobs:
       - name: install papermill
         if: ${{ env.RUN_NB_TESTS == '1' }}
         shell: bash -l {0}
-        run: pip install papermill[all]
+        run: |
+          conda install ipykernel
```

Comment:
```suggestion
          conda install ipykernel papermill
```

Reply:
Not exactly, but nearly. We need to specify -c conda-forge. Done.